db/migrate/20240429130154_remove_user_permissions_with_deleted_permission.rb

```diff
@@ -0,0 +1,27 @@
+class RemoveUserPermissionsWithDeletedPermission < ActiveRecord::Migration[7.1]
+  def change
+    return unless SupportedPermission.find_by(id: 1152).nil?
+
+    user_application_permissions_to_destroy = UserApplicationPermission.where(supported_permission_id: 2316)
```

Comment:
Should this be 1152?

```suggestion
    user_application_permissions_to_destroy = UserApplicationPermission.where(supported_permission_id: 1152)
```

It may be better to assign this to a variable and then the variable can be used in the three places the ID is referred to?